### PR TITLE
Fix syntax errors and punctuation

### PR DESCRIPTION
**Pull request by mlimber**

### CODE REVIEW

File: templates/prism/styles/docfx.vendor.css [hunk 1]
```diff
@@ -1454,7 +1454,8 @@ hr {
 }
 
 p {
```
Comment (brianlagunas) [inline templates/prism/styles/docfx.vendor.css]: this is no longer needed as the website has a new design which uses 100% of the available width

Comment (mlimber) [reply templates/prism/styles/docfx.vendor.css]: See the discussion, e.g., [here](https://www.freshconsulting.com/uiux-principle-46-text-box-width-should-help-users-read/) and [here](https://ux.stackexchange.com/questions/14928/why-do-websites-not-use-entire-width-of-browser) on why using the full width is not preferable from a UX perspective.

That said, I can revert that change if you want to take the others only.

Comment (brianlagunas) [reply templates/prism/styles/docfx.vendor.css]: The doc site responds properly to resizing and small devices. This is not need. Plus, this is not where you would make this change. All modifications should go in the main.css

Comment (mlimber) [reply templates/prism/styles/docfx.vendor.css]: It's not resizing and small devices I'm concerned with. It's that the UX is better for wide-screen if you limit the text width to 600-700px. Compare (extra width added for emphasis, but you get the idea):

Current UX

Proposed UX

I'm not claiming to say this design is good or even adequate (e.g., ideally the right and left columns would be up against the 700px, not way across the screen), but as far as readability goes on wide devices, it is better. 

I'm willing to drop this css change or move it to main.css. Your call.

Comment (brianlagunas) [reply templates/prism/styles/docfx.vendor.css]: point taken. I'll revisit the styling

Comment (mlimber) [reply templates/prism/styles/docfx.vendor.css]: Ok thanks. Let me know if you want me to change anything in this PR.

Comment (brianlagunas) [reply templates/prism/styles/docfx.vendor.css]: I fixed the styling, so you should remove this portion of your PR.

**Comment by mlimber** [comment]
@brianlagunas: Saw your change. Nice! I've reverted mine to the CSS.

**Review by brianlagunas — APPROVED** [review]
thanks

**Pull request merged**